protoc-gen-jersey/src/main/java/com/fullcontact/rpc/jersey/yaml/YamlHttpConfig.java

```diff
@@ -0,0 +1,43 @@
+package com.fullcontact.rpc.jersey.yaml;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * Created by kylehansen @Sypticus on 12/28/16.
```

Comment:
Lets make this javadoc relevant -- a good candidate would be at least a link to the doc page mentioning the YAML format.